sql-tumble-window-test/pom.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>sql-tumble-window-test</artifactId>
+
+    <parent>
+        <artifactId>hazelcast-jet-ansible-tests</artifactId>
+        <groupId>com.hazelcast.jet.tests</groupId>
+        <version>5.3.0-SNAPSHOT</version>
+    </parent>
+
+    <properties>
+        <main.basedir>${project.parent.basedir}</main.basedir>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.hazelcast.jet.tests</groupId>
+            <artifactId>soak-tests-common</artifactId>
+            <version>${hazelcast.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.hazelcast.jet.tests</groupId>
+            <artifactId>soak-tests-common-sql</artifactId>
+            <version>${hazelcast.version}</version>
+        </dependency>
+        <dependency>
```

Comment:
`soak-tests-common-sql` is required for `com.hazelcast.jet.tests.common.sql.TestRecordProducer` used by `AbstractTumbleWindowTest`. `hazelcast-jet-kafka` dependency looks to be unnecessary.

Reply:
My fault, I didn't thought `soak-tests-common-sql` but `hazelcast-sql` (just copy-pasted error on my side). Basically it's used just for two `SqlConnector` constants. Otherwise it seems not to be needed on client side I think.